extensions/search/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "schema.json#",
+  "title": "Search Extension",
+  "type": "object",
+  "description": "STAC Search Extension",
+  "additionalProperties": true,
+  "allOf": [
+    {
+      "$ref": "#/definitions/search_meta"
+    }
+  ],
+  "definitions": {
+    "search_meta": {
+      "properties": {
+        "properties": {
+          "type": "object",
+          "required": [
+            "next",
+            "returned"
+          ],
+          "properties": {
+            "next": {
+              "type": ["string", "null"]
+            },
+            "limit": {
+              "type": ["number", "null"]
+            },
+            "found": {
+              "type": "number"
+            },
+            "returned": {
```

Comment:
What is returned useful for? Isn't that just a `Array.size(features)` (or however array length is determined in your programming language of choice)?

Reply:
Correct.  The general consensus during discussions at Sprint 4 was to include this, even though it's something that can be computed.

Reply:
Okay, fine with that. It's just an extension, just trying to not clutter the spec too much. Do you remember the reason for it?